.coveragerc

```diff
@@ -1,4 +1,4 @@
 [run]
 source = ooni
 [report]
-omit = ooni/nettests/*, ooni/api/*, ooni/kit/*, ooni/contrib/*
+omit = ooni/nettests/*, ooni/api/*, ooni/kit/*, ooni/contrib/* Dockerfile docker-compose.yml README.rst
```

Comment:
I think you don't need to add text files to test `.coveragerc`. Since those are not source files, it will already be ignored by `coverage.py`.

Reply:
makes sense. will add a commit for that